### ISSUE TITLE
Beautify the About Section and developers section

### ISSUE THREAD

**Issue by sau-mili** [mindspace-prototype/MindSpace#156]
I can modify the UI of the about section and developers section of the page. Please assign me this

**Comment by 761jayanth8** [comment]
Yes Sir I completely understand your problem statement and I have fixed this one Sir and here are the images of both small and medium devices about us section. sir if you like it please assign me as contributor.
![Screenshot 2024-05-11 021859](https://github.com/The-MindSpace/MindSpace-Web/assets/136685608/3cc7a6a3-778a-432d-a53e-971d0c410069)
![Screenshot 2024-05-11 021917](https://github.com/The-MindSpace/MindSpace-Web/assets/136685608/e51328a7-7287-4437-8049-badc372206d6)

**Comment by Anou25** [comment]
Hi, I would like to work on this issue, please can you assign this to me.

**Comment by Paavni-Goyal** [comment]
Hey, I would like to work on the same. Can you please assign this to me under GSSoC'24.
